src/openforms/forms/api/viewsets.py

```diff
@@ -608,6 +609,29 @@ def logic_rules_list(self, request, *args, **kwargs):
         )
         return Response(serializer.data, status=status.HTTP_200_OK)
 
+    @extend_schema(
+        summary=_("JSON schema"),
+        description=_("Generate the JSON schema for a form."),
+        parameters=[UUID_OR_SLUG_PARAMETER],
+        request=None,
+        responses={
+            (
+                status.HTTP_200_OK,
+                "application/json",
+            ): OpenApiTypes.OBJECT
+        },
```

Comment:
I think you can omit the `application/json`, since the serialization itself is handled by DRF renderers.